Mathlib/Order/WellFoundedSet.lean

```diff
@@ -234,69 +245,71 @@ theorem isWF_iff_no_descending_seq :
 
 end Preorder
 
-/-!
-### Partially well-ordered sets
-
-A set is partially well-ordered by a relation `r` when any infinite sequence contains two elements
-where the first is related to the second by `r`. Equivalently, any antichain (see `IsAntichain`) is
-finite, see `Set.partiallyWellOrderedOn_iff_finite_antichains`.
```

Comment:
Let's preserve this docstring somewhere: it's more useful to see an actual definition than a reference to another definition in a comment for a definition